plugins/org.eclipse.n4js/src/org/eclipse/n4js/resource/N4JSResource.java

```diff
@@ -810,6 +897,9 @@ public EObject doResolveProxy(InternalEObject proxy, EObject objectContext) {
 	 * @return true, if the referenced object may come from the the index TModule.
 	 */
 	private boolean canResolveFromDescription(URI targetUri) {
+		if (targetUri.fragment().startsWith("/0")) {
+			return false;
+		}
 		return canLoadFromDescriptionHelper.canLoadFromDescription(targetUri.trimFragment(), getResourceSet());
```

Comment:
canLoadFromDescriptionHelper accepts a resourceURI as parameter. We would need to ensure that all callers always have a fragment at hand. Preferably I'd leave this method here. Calling it canLoadFromDescription instead of canResolve.. is fine for me.